modules/oe_showcase_list_pages/tests/src/ExistingSite/ListPagesTest.php

```diff
@@ -83,24 +83,15 @@ public function testCreateListPages() {
     $this->indexItems('oe_list_pages_index');
 
     // Create the News listing page.
-    $this->drupalGet('node/add/oe_list_page');
-    $page->fillField('Title', 'News list page');
-    $page->fillField('Source entity type', 'node');
-    $page->fillField('Source bundle', 'oe_sc_news');
-    $page->pressButton('Save');
-
-    $node = $this->getNodeByTitle('News list page');
-    $this->drupalGet('node/' . $node->id() . '/edit');
-    $page->checkField('Override default exposed filters');
-    $page->checkField('emr_plugins_oe_list_page[wrapper][exposed_filters][oelp_oe_sc_news__title]');
-    $page->checkField('emr_plugins_oe_list_page[wrapper][exposed_filters][oelp_oe_sc_news__oe_publication_date]');
-    $page->pressButton('Save');
-
-    $this->drupalGet('node/' . $node->id());
+    $list_page = $this->createListPage('oe_sc_news', [
+      'oelp_oe_sc_news__title' => 'oelp_oe_sc_news__title',
+      'oelp_oe_sc_news__oe_publication_date' => 'oelp_oe_sc_news__oe_publication_date',
```

Comment:
Simplify as suggested in other comments below:
```suggestion
      'oelp_oe_sc_news__title',
      'oelp_oe_sc_news__oe_publication_date',
```